README.md

```diff
@@ -14,9 +14,51 @@ Within vim, press Space and Backspace (in normal mode) to move forwards and back
 
 Especially useful for live demos with a lot of canned code, which you want to present and run incrementally.
 
+You only need one file. That file will be your slides (notice the plural). Each
+change and save will create a new commit. Moving back and forth between slides
+will checkout the current working directory to the previous commit and next
+commit respectively and reload the buffer in vim.
```

Comment:
Ah, it's a good idea to clarify this. But "checkout the current working directory" sounds wrong, have you seen this phrasing used anywhere else? I usually say "checkout the commit", not "checkout the current working directory to the commit".

Reply:
Changed it.